cmd/reposervercontroller/main.go

```diff
@@ -64,22 +62,15 @@ func main() {
 	var metricsAddr string
 	var probeAddr string
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
-	logLevel := getLogLevel()
-
-	opts := zap.Options{
-		Level: logLevel,
-	}
 
 	// Disable metrics server
 	metricsAddr = "0"
-	opts.BindFlags(flag.CommandLine)
 	flag.Parse()
-	logger := zap.New(zap.UseFlagOptions(&opts))
-	ctrl.SetLogger(logger)
+	ctrl.SetLogger(utils.GetLogger())
```

Comment:
We should also avoid making unrelated changes to K8s upgrade in the same PR. Unrelated improvements can be done as a follow-up PR.
Added more comments on the `GetLogger` method below

Reply:
Reverted/